app/assets/stylesheets/components/_new_post_form.scss

```
@@ -0,0 +1,183 @@
+.new_post {
+  align-items: center;
+  background-color: $white-light;
+  display: flex;
+  flex-direction: column;
+}
+
+.new_post__view {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+}
```

You can use sass sintax here using &
Example:
```suggestion
.new_post {
  // content
  &__view {
    // content
  }
}
```

---

Done, thank you